### ISSUE TITLE
Style Guide Formatting

### ISSUE THREAD

**Issue by pamaph20** [kevinaangstadt/CS340-S23-Notes#126]
Within the style guide it states to have all major sections of notes separated out with horizontal lines -> 
"Major sections should be separated by horizontal rules"
However, there are some instances where we define major aspects of a process but do not go into depth about it. Without the in depth portion the maintaining of the horizontal rules makes the markdown file feel clunky (Reference down below) . How would we go about this to make the file look more presentable.

## 4) something here
>#### stuff

* more stuff
* things

---
## 5) Another subsection
---
## 6) Another section
(See also [Markdown Guide]((See also [Markdown Guide - Horizontal Rules](https://www.youtube.com/watch?v=xvFZjo5PgG0&ab_channel=Duran))

**Comment by mattabell03** [comment]
Maybe instead of a full page-width of horizontal lines between section headers with no/little content we could just do a blank line or some other smaller distinguishing symbol like ***?

**Comment by edvinStra** [comment]
I agree with @mattabell03, maybe a blank line and start the new subsection with bigger font and bolded as a sub-header?
